src/main/java/be/seeseemelk/mockbukkit/inventory/ItemStackMock.java

```diff
@@ -250,6 +273,11 @@ else if (args.containsKey("meta"))
 		return result;
 	}
 
+	public InteractionResult simulateUseItemOn(PlayerMock playerMock, Location clickedPos, EquipmentSlot hand)
+	{
+		throw new UnimplementedOperationException();
+	}
```

Comment:
Why are you adding a placeholder method here?

It does not add any feature to mockbukkit, It's completely unnecessary.